.github/workflows/continuous-integration-workflow.yml

```diff
@@ -55,7 +55,23 @@ jobs:
         name: Build docker image
         working-directory: common/jenkins-agents/nodejs12/docker
         run: |
-            docker build --tag agent-nodejs12-test \
+            docker build --tag agent-nodejs12-test --file Dockerfile.rhel7 \

```

Comment:
I don't know if it matters, but we used to add a suffix to the tag with each version for the go agent.

Reply:
Changed in 4aeeeb6.